Assets/Game/Addons/UnityConsole/Console/Scripts/DefaultCommands.cs

```diff
@@ -643,9 +643,10 @@ public static string Execute(params string[] args)
         private static class SetWalkSpeed
         {
             public static readonly string name = "set_walkspeed";
+            public static readonly string uid = "speedmod_uid";
```

Comment:
What's the purpose of these lines? I can't see where it's referred from. Is this for mods?

Reply:
That can be removed. It was an addition I forgot about. It is not needed.